### PR TITLE
Allow 2 lines for user name on intro screen

### LINKED ISSUE #9893: Allow names in onboarding to use two lines to fit really long ones
It's really important not to shorten the name on this step and to show the whole thing. It's the moment when the user should choose the name they would not be able to change later.

Here's how the long names should look like:

Screen Shot 2020-01-23 at 17 23 29

Please note:
- The picture stays where it is;
- The radio button is vertically centered;

Figma: https://www.figma.com/file/dEIljL7UPbXgsZUA0Q4qlE5E/Onboarding?node-id=4540%3A11551

### PR DESCRIPTION
**Pull request by Ferossgp**
Fixes #9893

### CODE REVIEW

File: src/status_im/ui/screens/intro/views.cljs [hunk 1]
```diff
@@ -119,39 +119,40 @@
...
-(defn choose-key [{:keys [multiaccounts selected-id view-height]}]
+(defn choose-key [{:keys [multiaccounts selected-id]}]
   [react/view
    {:style {:flex            1
             :justify-content :center}}
    [react/view
-    {:style {:flex       1
-             :height     200
-             :max-height 338}}
+    {:style {:height 410}}
```
Comment (flexsurfer) [inline src/status_im/ui/screens/intro/views.cljs]: are you sure this will work on small screens?

Comment (Ferossgp) [reply src/status_im/ui/screens/intro/views.cljs]: Tested only on iPhone 8, will modify to support smaller screens.

File: trace.edn [hunk 2]
```diff
@@ -0,0 +1,3118 @@
+{:log
```
Comment (flexsurfer) [inline trace.edn]: why this file is here?

Comment (flexsurfer) [reply trace.edn]: rebase needed?

**Comment by flexsurfer** [comment]
@Ferossgp does it support small screens now?

File: trace.edn (hunk 2 above)
Comment (Ferossgp) [reply trace.edn]: Yes, it will be ignored after rebase

**Comment by Ferossgp** [comment]
@flexsurfer yes, it will be scrollable on small screens

**Comment by flexsurfer** [comment]
@Ferossgp what the status of this PR  ? could you rebase it and merge conflicts so we can test and merge it ?

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**